bettermdptools/algorithms/rl.py

```diff
@@ -44,6 +45,10 @@ def __init__(self, env):
         #       return np.random.choice(indxs)
         #   else:
         #       return np.random.randint(len(Q[state]))
+        seed = get_seed()
+        if seed is not None:
+            np.random.seed(seed)
+            env.reset(seed=seed)
```

Comment:
I think this should actually be put in the initialization of each of the environment wrappers right? and _also_ wherever `env.reset` is currently called.

Reply:
I put the seed functionality in `test_env()` and `RL()` because if I initialize the wrapper with seed then run VI, PI, and QL, the environment only resets for VI (mean test_scores are `-0.13 , -0.01, -0.12`, respectively). Putting the seed in `test_env()` resets the env every time each algorithm is called (mean test_scores are `-0.13, -0.13, and -0.1`). 

Regarding your suggestion to set the seed in `env.reset()`, are you suggesting that every time `env.reset()` is called, the seed should automatically be set based on the global seed, like `env.reset(seed=SEED)`? If so, could you provide some guidance on how I might go about ensuring this?

As of my understanding, I only call `env.reset()` once before running each algorithm, and this is already handled with the seed implementation I added in `test_env()` and `RL()`. In the current code, the only place I see `env.reset()` being explicitly called is in the q_learning function: `state, info = self.env.reset()`. However, this doesn’t seem to directly allow for setting the seed in this context.

Thank you!